app/javascript/components/GameBoardSetter.js

```diff
@@ -18,32 +45,33 @@ const GameBoardSetter = (props) => {
     })
   }
 
-  useEffect(() => {
-    setLastEventId(props.lastEventId);
-    setCards(props.cards);
-    setSetters({
-      setLocations: (l) => {
-        setLocations(locations.map((location) => {
-          if(location.id === l.player_id) {
-            return {...location, [location.name]: l.player_name }
-          } else { return location }
-        }));
-        return locations.filter(l => l.type === 'player').map(l => l.id).indexOf(l.player_id) + 1;
-      },
-      location: locations
-    });
-    return () => {
-      setCards([]);
-      setSetters({setLocations: () => {}, locations: [], lastEventId: () => {}, setlastEventId: () => {}})
-    }
-  });
-
-  return <GameBoard
-    name={props.name}
-    locations={locations}
-    stacks={props.stacks}
-    params={props.params}
-  />
+  // useEffect(() => {
```

Comment:
This needs to be added back to get the tests to work as this functionality has not yet been moved